defects4cpp.errors.exception.DppTaxonomyNotFoundError
=====================================================

.. currentmodule:: defects4cpp.errors.exception

.. autoexception:: DppTaxonomyNotFoundError